addons/sourcemod/scripting/include/utilshelper.inc

```diff
@@ -259,10 +259,13 @@ stock int GetEdictsCount()
 
 stock void StringToLowerCase(char[] input)
 {
-    for (int i = 0; i < strlen(input); i++)
-    {
-        input[i] = CharToLower(input[i]);
-    }
+	int i, x;
+	while ((x = input[i]) != 0)
```

Comment:
!= '\0'
so that we understand that its the null terminator

assign value at the end of while loop for clarity

Reply:
Like this ?

Reply:
yes but i was talking about putting this:
```
x = input[i]
```

outside the while condition because its a bad practice to do this